### PR TITLE
Add page validation and configure travis job

### PR DESCRIPTION
**Pull request by qlmmlp**
https://spryker.atlassian.net/browse/TE-8782

- Create ruby gem for validation functionality 
- Add CLI command plugin to Jekyll
- Integrate command in CI

TODO: gem source code might be placed in separate repo.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,11 +2,21 @@ language: ruby
...
+
+jobs:
+  include:
+    - name: Page validation
+      script:
+        - bundle exec jekyll validate
+      before_install:
+        - gem install bundler
+
+    - name: Push pages to Algolia
```
Comment (demkos) [inline .travis.yml:17]: This build must work with master branch only

File: Gemfile [hunk 2]
```diff
@@ -1,7 +1,6 @@
 source "https://rubygems.org"
 
 # to publish on github page
-gem 'github-pages', group: :jekyll_plugins
```
Comment (demkos) [inline Gemfile:4]: You removed `github-pages` but `jekyll` was not enabled. How is it works?

File: _plugins/spryker_documentation/Gemfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
```
Comment (demkos) [inline _plugins/spryker_documentation/Gemfile:1]: Why do you need a custom gem?

File: Gemfile.lock [hunk 4]
```diff
@@ -1,12 +1,11 @@
+PATH
```
Comment (demkos) [inline Gemfile.lock:1]: This lock file contains a too big amount of changes. I don't any reason for doing such changes.

**Review by demkos — CHANGES_REQUESTED** [review]

File: _plugins/spryker_documentation/Gemfile (hunk 3 above)
Comment (qlmmlp) [reply _plugins/spryker_documentation/Gemfile:1]: There is no other way for custom command plugin to be integrated only as a new gem. CLI not added if it just placed inside `_plugins` as `.rb` script.

File: .travis.yml (hunk 1 above)
Comment (qlmmlp) [reply .travis.yml:17]: Adressed.

File: Gemfile.lock (hunk 4 above)
Comment (qlmmlp) [reply Gemfile.lock:1]: Addressed

**Review by demkos — APPROVED** [review]

**Pull request merged**